payments_config/__init__.py

```diff
@@ -62,6 +63,10 @@ def __init__(self, seller, config):
             raise ValueError('Recurrence should be not set or monthly: {}'.
                              format(self.recurrence))
 
+        if (self.user_identification not in self.valid_user_identifications):
```

Comment:
aha, nevermind. I didn't realize there was already validation when it's not defined at all. That's better.